edl/oecore.edl

```diff
@@ -4,7 +4,7 @@
 /*
 **==============================================================================
 **
-** tee.edl:
+** oecore.edl:
```

Comment:
Maybe consider changing this to just core.edl. Then if the user's edl search path is <install_path>/include their import line would be: `from "openenclave/edl/core.edl" import *` which seems a little cleaner